src/index.html

```diff
@@ -6,13 +6,52 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="container">
+      <div class="card">
+
+        <img
```

Comment:
Don't add a blank line between parent and child elements.
**To add some "air" and simplify reading. But don't add them between parent and child elements.**
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/cb81e479-f086-47a1-b710-f613f01a4f21)